packages/visualizations/src/components/MarkdownText/MarkdownText.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts" context="module">
+
+</script>
+
+<script lang="ts">
+    import type {
+        MarkdownTextOptions,
+    } from '../types';
+    import type { Async } from '../../types';
+    export let data: Async<string>;
+    export let options: MarkdownTextOptions;
+    var MarkdownIt = require('markdown-it');
```

Comment:
I didn't manage to make it work properly.

For example with:
```
import * as MarkdownIt from 'markdown-it';
```

the resulting `dist/index.js` contains a `require('./lib/')` which fails because there is no such folder in `dist`. I think we may be missing some plugin to deal with non-ES6 packages, but my knowledge is very limited, so I went with what worked immediately 😬